src/components/Footer/Footer.spec.js

```diff
@@ -0,0 +1,12 @@
+import {render, screen} from '@testing-library/react';
+import {BrowserRouter} from 'react-router-dom';
+
+import Footer from './Footer';
+
+describe('Footer', () => {
+  it('renders footer', () => {
+    render(<Footer />, {wrapper: BrowserRouter});
+    const footerElement = screen.getByText('Search');
```

Comment:
I think you should be able to query this by using `screen.getByRole('link', {name: 'Search})`. That would be a bit more precise and automatically tests the accessibility of your element.

Reply:
Thanks! Will implement, still getting used to testing...